admin/code/tce_page_info.php

```diff
@@ -147,7 +147,7 @@
 echo '<li>[HI] Hindi : Mahesh K Bhandari, Shekhar K Maravi, Pradeep K Nayak</li>'.K_NEWLINE;
 echo '<li>[HE] Hebrew : Oron Peled</li>'.K_NEWLINE;
 echo '<li>[HU] Hungarian : Peter Ivanyi, Tibor Balázs</li>'.K_NEWLINE;
-echo '<li>[ID] Indonesian : Ahmad Bardosono</li>'.K_NEWLINE;
+echo '<li>[ID] Indonesian : Ahmad Bardosono, Maman Sulaeman</li>'.K_NEWLINE;
```

Comment:
I am not sure why this change.
Have you contributed to the Indonesian translation?

Reply:
Yes, I have made a contribution in Indonesian Translation. Here is my merged pull request https://github.com/tecnickcom/tcexam/pull/314/files